### PR TITLE
Delete FunctionsJava

### PR DESCRIPTION
**Pull request by pragnagopa**
- Updated Java Worker to 1.5.0: https://github.com/Azure/azure-functions-java-worker/releases/tag/1.5.0
- User profiles are loaded for placeholders which eliminates latency issues when starting java.exe

### CODE REVIEW

**Comment by pragnagopa** [comment]
cc @kranthikumarm as fyi

**Comment by pragnagopa** [comment]
ping! - @mhoeger  / @fabiocav  can you please take a look? this needs to go into the next release

File: src/WebJobs.Script/Rpc/Configuration/WorkerConfigFactory.cs [hunk 1]
```diff
@@ -131,14 +129,6 @@ internal void AddProvider(string workerDir)
                 var languageSection = _config.GetSection($"{LanguageWorkerConstants.LanguageWorkersSectionName}:{workerDescription.Language}");
                 workerDescription.Arguments = workerDescription.Arguments ?? new List<string>();
 
-                descriptionProfiles = GetWorkerDescriptionProfiles(workerConfig);
-                if (ScriptSettingsManager.Instance.IsAppServiceEnvironment)
-                {
-                    // Overwrite default Description with AppServiceEnv profile
-                    // TODO:pgopa delete after ANT78
-                    workerDescription = GetWorkerDescriptionFromProfiles(LanguageWorkerConstants.WorkerDescriptionAppServiceEnvProfileName, descriptionProfiles, workerDescription);
```
Comment (mhoeger) [inline src/WebJobs.Script/Rpc/Configuration/WorkerConfigFactory.cs]: Can we also remove `WorkerDescriptionAppServiceEnvProfileName` from LangageWorkerConstants?

Comment (pragnagopa) [reply src/WebJobs.Script/Rpc/Configuration/WorkerConfigFactory.cs]: thanks! fixed.

Comment (pragnagopa) [reply src/WebJobs.Script/Rpc/Configuration/WorkerConfigFactory.cs]: And updated the description. ANT83 has fix that removes dependency on FunctionsJava

**Pull request merged**